### ISSUE TITLE
[Bug]: [Nightly]Fail to connect reporting `Handshake failed with fatal error SSL .. WRONG_VERSION_NUMBER.`

### ISSUE THREAD

**Issue by NicoYuan1986** [milvus-io/milvus#24469]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Environment

```markdown
- Milvus version: 76c24db
- Deployment mode(standalone or cluster): cluster
- MQ type(rocksmq, pulsar or kafka):     kafka
- SDK version(e.g. pymilvus v2.0.0rc2): 2.2.9.dev16
- OS(Ubuntu or CentOS): 
- CPU/Memory: 
- GPU: 
- Others:
```

### Current Behavior

Fail to connect reporting `Handshake failed with fatal error SSL .. WRONG_VERSION_NUMBER.`
```
[2023-05-28T21:30:35.727Z] ----------------------------- Captured stderr call -----------------------------
[2023-05-28T21:30:35.727Z] E0528 21:10:41.706533142   30848 ssl_transport_security.cc:1420]       Handshake failed with fatal error SSL_ERROR_SSL: error:100000f7:SSL routines:OPENSSL_internal:WRONG_VERSION_NUMBER.
[2023-05-28T21:30:35.727Z] E0528 21:10:42.701474705   25064 ssl_transport_security.cc:1420]       Handshake failed with fatal error SSL_ERROR_SSL: error:100000f7:SSL routines:OPENSSL_internal:WRONG_VERSION_NUMBER.
[2023-05-28T21:30:35.727Z] E0528 21:10:44.147112552  131743 ssl_transport_security.cc:1420]       Handshake failed with fatal error SSL_ERROR_SSL: error:100000f7:SSL routines:OPENSSL_internal:WRONG_VERSION_NUMBER.
[2023-05-28T21:30:35.727Z] E0528 21:10:46.403843006   26224 ssl_transport_security.cc:1420]       Handshake failed with fatal error SSL_ERROR_SSL: error:100000f7:SSL routines:OPENSSL_internal:WRONG_VERSION_NUMBER.
[2023-05-28T21:30:35.727Z] E0528 21:10:50.549870742   25823 ssl_transport_security.cc:1420]       Handshake failed with fatal error SSL_ERROR_SSL: error:100000f7:SSL routines:OPENSSL_internal:WRONG_VERSION_NUMBER.
[2023-05-28T21:30:35.727Z] ------------------------------ Captured log call -------------------------------
[2023-05-28T21:30:35.727Z] [2023-05-28 21:10:41 - DEBUG - ci_test]: (api_request)  : [Connections.connect] args: ['default', 'root', 'Milvus', ''], kwargs: {'host': 'mdk-204-n-milvus.milvus-ci', 'port': '19530'} (api_request.py:56)
[2023-05-28T21:30:35.727Z] [2023-05-28 21:10:51 - ERROR - ci_test]: Traceback (most recent call last):
[2023-05-28T21:30:35.727Z]   File "/usr/local/lib/python3.8/site-packages/pymilvus/client/grpc_handler.py", line 119, in _wait_for_channel_ready
[2023-05-28T21:30:35.727Z]     grpc.channel_ready_future(self._channel).result(timeout=timeout)
[2023-05-28T21:30:35.727Z]   File "/usr/local/lib/python3.8/site-packages/grpc/_utilities.py", line 151, in result
[2023-05-28T21:30:35.727Z]     self._block(timeout)
[2023-05-28T21:30:35.727Z]   File "/usr/local/lib/python3.8/site-packages/grpc/_utilities.py", line 97, in _block
[2023-05-28T21:30:35.727Z]     raise grpc.FutureTimeoutError()
[2023-05-28T21:30:35.727Z] grpc.FutureTimeoutError
[2023-05-28T21:30:35.727Z] 
[2023-05-28T21:30:35.727Z] The above exception was the direct cause of the following exception:
[2023-05-28T21:30:35.727Z] 
[2023-05-28T21:30:35.727Z] Traceback (most recent call last):
[2023-05-28T21:30:35.727Z]   File "/home/jenkins/agent/workspace/tests/python_client/utils/api_request.py", line 26, in inner_wrapper
[2023-05-28T21:30:35.727Z]     res = func(*args, **_kwargs)
[2023-05-28T21:30:35.727Z]   File "/home/jenkins/agent/workspace/tests/python_client/utils/api_request.py", line 57, in api_request
[2023-05-28T21:30:35.727Z]     return func(*arg, **kwargs)
[2023-05-28T21:30:35.727Z]   File "/usr/local/lib/python3.8/site-packages/pymilvus/orm/connections.py", line 332, in connect
[2023-05-28T21:30:35.727Z]     connect_milvus(**kwargs, user=user, password=password, db_name=db_name)
[2023-05-28T21:30:35.727Z]   File "/usr/local/lib/python3.8/site-packages/pymilvus/orm/connections.py", line 276, in connect_milvus
[2023-05-28T21:30:35.727Z]     gh._wait_for_channel_ready(timeout=timeout)
[2023-05-28T21:30:35.727Z]   File "/usr/local/lib/python3.8/site-packages/pymilvus/client/grpc_handler.py", line 123, in _wait_for_channel_ready
[2023-05-28T21:30:35.727Z]     raise MilvusException(Status.CONNECT_FAILED,
[2023-05-28T21:30:35.727Z] pymilvus.exceptions.MilvusException: <MilvusException: (code=2, message=Fail connecting to server on mdk-204-n-milvus.milvus-ci:19530. Timeout)>
[2023-05-28T21:30:35.727Z]  (api_request.py:39)
[2023-05-28T21:30:35.727Z] [2023-05-28 21:10:51 - ERROR - ci_test]: (api_response) : <MilvusException: (code=2, message=Fail connecting to server on mdk-204-n-milvus.milvus-ci:19530. Timeout)> (api_request.py:40)
```

### Expected Behavior

pass

### Milvus Log

1. link: https://jenkins.milvus.io:18080/blue/organizations/jenkins/Milvus%20Nightly%20CI/detail/2.2.0/204/pipeline/144
2. log: [artifacts-milvus-distributed-kafka-nightly-204-pymilvus-e2e-logs.tar.gz](https://jenkins.milvus.io:18080/job/Milvus%20Nightly%20CI/job/2.2.0/204/artifact/artifacts-milvus-distributed-kafka-nightly-204-pymilvus-e2e-logs.tar.gz)
3. failed time: [2023-05-28T21:11:02.691Z] [gw4] [ 97%] FAILED testcases/test_utility.py::TestUtilityRBAC::test_revoke_user_after_delete_user 
[2023-05-28T21:11:02.948Z] [gw4] [ 97%] ERROR testcases/test_utility.py::TestUtilityRBAC::test_revoke_user_after_delete_user

**Comment by wakakawaka** [comment]
Encountered the same problem


```
E0529 13:42:59.964674343     212 ssl_transport_security.cc:1495] Handshake failed with fatal error SSL_ERROR_SSL: error:100000f7:SSL routines:OPENSSL_internal:WRONG_VERSION_NUMBER.
E0529 13:43:00.958081810     212 ssl_transport_security.cc:1495] Handshake failed with fatal error SSL_ERROR_SSL: error:100000f7:SSL routines:OPENSSL_internal:WRONG_VERSION_NUMBER.
E0529 13:43:02.447099754     212 ssl_transport_security.cc:1495] Handshake failed with fatal error SSL_ERROR_SSL: error:100000f7:SSL routines:OPENSSL_internal:WRONG_VERSION_NUMBER.
E0529 13:43:04.802805824     212 ssl_transport_security.cc:1495] Handshake failed with fatal error SSL_ERROR_SSL: error:100000f7:SSL routines:OPENSSL_internal:WRONG_VERSION_NUMBER.
E0529 13:43:08.184160754     212 ssl_transport_security.cc:1495] Handshake failed with fatal error SSL_ERROR_SSL: error:100000f7:SSL routines:OPENSSL_internal:WRONG_VERSION_NUMBER.
E0529 13:43:09.975334938     218 ssl_transport_security.cc:1495] Handshake failed with fatal error SSL_ERROR_SSL: error:100000f7:SSL routines:OPENSSL_internal:WRONG_VERSION_NUMBER.
E0529 13:43:10.975154351     218 ssl_transport_security.cc:1495] Handshake failed with fatal error SSL_ERROR_SSL: error:100000f7:SSL routines:OPENSSL_internal:WRONG_VERSION_NUMBER.
E0529 13:43:12.793673465     212 ssl_transport_security.cc:1495] Handshake failed with fatal error SSL_ERROR_SSL: error:100000f7:SSL routines:OPENSSL_internal:WRONG_VERSION_NUMBER.
E0529 13:43:15.117362537     212 ssl_transport_security.cc:1495] Handshake failed with fatal error SSL_ERROR_SSL: error:100000f7:SSL routines:OPENSSL_internal:WRONG_VERSION_NUMBER.
E0529 13:43:15.468288378     212 ssl_transport_security.cc:1495] Handshake failed with fatal error SSL_ERROR_SSL: error:100000f7:SSL routines:OPENSSL_internal:WRONG_VERSION_NUMBER.
E0529 13:43:19.203744547     212 ssl_transport_security.cc:1495] Handshake failed with fatal error SSL_ERROR_SSL: error:100000f7:SSL routines:OPENSSL_internal:WRONG_VERSION_NUMBER.
Exception in thread django-main-thread:
Traceback (most recent call last):
  File "/usr/local/lib/python3.7/threading.py", line 926, in _bootstrap_inner
    self.run()
  File "/usr/local/lib/python3.7/threading.py", line 870, in run
    self._target(*self._args, **self._kwargs)
  File "/usr/local/lib/python3.7/site-packages/django/utils/autoreload.py", line 64, in wrapper
    fn(*args, **kwargs)
  File "/usr/local/lib/python3.7/site-packages/django/core/management/commands/runserver.py", line 118, in inner_run
    self.check(display_num_errors=True)
  File "/usr/local/lib/python3.7/site-packages/django/core/management/base.py", line 423, in check
    databases=databases,
  File "/usr/local/lib/python3.7/site-packages/django/core/checks/registry.py", line 76, in run_checks
    new_errors = check(app_configs=app_configs, databases=databases)
  File "/usr/local/lib/python3.7/site-packages/django/core/checks/urls.py", line 13, in check_url_config
    return check_resolver(resolver)
  File "/usr/local/lib/python3.7/site-packages/django/core/checks/urls.py", line 23, in check_resolver
    return check_method()
  File "/usr/local/lib/python3.7/site-packages/django/urls/resolvers.py", line 416, in check
    for pattern in self.url_patterns:
  File "/usr/local/lib/python3.7/site-packages/django/utils/functional.py", line 48, in __get__
    res = instance.__dict__[self.name] = self.func(instance)
  File "/usr/local/lib/python3.7/site-packages/django/urls/resolvers.py", line 602, in url_patterns
    patterns = getattr(self.urlconf_module, "urlpatterns", self.urlconf_module)
  File "/usr/local/lib/python3.7/site-packages/django/utils/functional.py", line 48, in __get__
    res = instance.__dict__[self.name] = self.func(instance)
  File "/usr/local/lib/python3.7/site-packages/django/urls/resolvers.py", line 595, in urlconf_module
    return import_module(self.urlconf_name)
  File "/usr/local/lib/python3.7/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1006, in _gcd_import
  File "<frozen importlib._bootstrap>", line 983, in _find_and_load
  File "<frozen importlib._bootstrap>", line 967, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 677, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/work_dir/sim/urls.py", line 30, in <module>
    path("checktext/", include("checktext.urls")),
  File "/usr/local/lib/python3.7/site-packages/django/urls/conf.py", line 34, in include
    urlconf_module = import_module(urlconf_module)
  File "/usr/local/lib/python3.7/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1006, in _gcd_import
  File "<frozen importlib._bootstrap>", line 983, in _find_and_load
  File "<frozen importlib._bootstrap>", line 967, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 677, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/work_dir/checktext/urls.py", line 3, in <module>
    from . import views
  File "/work_dir/checktext/views.py", line 9, in <module>
    from checktext.sim_text import similarity
  File "/work_dir/checktext/sim_text/similarity.py", line 1, in <module>
    from checktext.sim_text.sim_towhee import towhee_simhash_64, towhee_vec, towhee_segment_simhash_64
  File "/work_dir/checktext/sim_text/sim_towhee/towhee_simhash_64.py", line 22, in <module>
    ops.ann_insert.milvus_client(host=HOST, port=PORT, user=USER, password=PASSWORD, collection_name=COLLECTION_NAME))
  File "/usr/local/lib/python3.7/site-packages/towhee/runtime/pipeline.py", line 116, in output
    run_pipe.preload()
  File "/usr/local/lib/python3.7/site-packages/towhee/runtime/runtime_pipeline.py", line 140, in preload
    return _Graph(self._dag_repr.nodes, self._dag_repr.edges, self._operator_pool, self._thread_pool)
  File "/usr/local/lib/python3.7/site-packages/towhee/runtime/runtime_pipeline.py", line 64, in __init__
    self.initialize()
  File "/usr/local/lib/python3.7/site-packages/towhee/runtime/runtime_pipeline.py", line 75, in initialize
    raise RuntimeError(node.err_msg)
RuntimeError: Node-ann-insert/milvus-client-2 runs failed, error msg: Create ann-insert/milvus-client-2 operator ann-insert/milvus-client:main with args None and kws {'host': 'milvus-test.cloud-search', 'port': 19530, 'user': 'dev', 'password': 'WVHGG7MjCdSz8ODdqwDN', 'collection_name': 'real_similarity_simhash_64'} failed, err: <MilvusException: (code=2, message=Fail connecting to server on milvus-test.cloud-search:19530. Timeout)>, Traceback (most recent call last):
  File "/usr/local/lib/python3.7/site-packages/pymilvus/client/grpc_handler.py", line 114, in _wait_for_channel_ready
    grpc.channel_ready_future(self._channel).result(timeout=timeout)
  File "/usr/local/lib/python3.7/site-packages/grpc/_utilities.py", line 139, in result
    self._block(timeout)
  File "/usr/local/lib/python3.7/site-packages/grpc/_utilities.py", line 85, in _block
    raise grpc.FutureTimeoutError()
grpc.FutureTimeoutError

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/usr/local/lib/python3.7/site-packages/towhee/runtime/nodes/node.py", line 93, in initialize
    self._node_repr.op_info.tag)
  File "/usr/local/lib/python3.7/site-packages/towhee/runtime/operator_manager/operator_pool.py", line 99, in acquire_op
    op = self._op_loader.load_operator(hub_op_id, op_args, op_kws, tag)
  File "/usr/local/lib/python3.7/site-packages/towhee/runtime/operator_manager/operator_loader.py", line 147, in load_operator
    op = factory(function, arg, kws, tag)
  File "/usr/local/lib/python3.7/site-packages/towhee/runtime/operator_manager/operator_loader.py", line 130, in _load_operator_from_hub
    return self._load_operator_from_path(path, function, arg, kws)
  File "/usr/local/lib/python3.7/site-packages/towhee/runtime/operator_manager/operator_loader.py", line 118, in _load_operator_from_path
    return self._instance_operator(op, arg, kws) if op is not None else None
  File "/usr/local/lib/python3.7/site-packages/towhee/runtime/operator_manager/operator_loader.py", line 156, in _instance_operator
    return op(*arg, **kws) if kws is not None else op(*arg)
  File "/root/.towhee/operators/ann-insert/milvus-client/versions/main/__init__.py", line 4, in milvus_client
    return MilvusClient(*args, **kwargs)
  File "/root/.towhee/operators/ann-insert/milvus-client/versions/main/milvus_client.py", line 31, in __init__
    user=user, password=password, secure=True)
  File "/usr/local/lib/python3.7/site-packages/pymilvus/orm/connections.py", line 303, in connect
    connect_milvus(**kwargs, user=user, password=password)
  File "/usr/local/lib/python3.7/site-packages/pymilvus/orm/connections.py", line 274, in connect_milvus
    gh._wait_for_channel_ready(timeout=timeout)
  File "/usr/local/lib/python3.7/site-packages/pymilvus/client/grpc_handler.py", line 118, in _wait_for_channel_ready
    f'Fail connecting to server on {self._address}. Timeout') from e
pymilvus.exceptions.MilvusException: <MilvusException: (code=2, message=Fail connecting to server on milvus-test.cloud-search:19530. Timeout)>


E0529 13:43:24.905099242     218 ssl_transport_security.cc:1495] Handshake failed with fatal error SSL_ERROR_SSL: error:100000f7:SSL routines:OPENSSL_internal:WRONG_VERSION_NUMBER.
E0529 13:43:25.729205751     212 ssl_transport_security.cc:1495] Handshake failed with fatal error SSL_ERROR_SSL: error:100000f7:SSL routines:OPENSSL_internal:WRONG_VERSION_NUMBER.
E0529 13:43:35.929113137     218 ssl_transport_security.cc:1495] Handshake failed with fatal error SSL_ERROR_SSL: error:100000f7:SSL routines:OPENSSL_internal:WRONG_VERSION_NUMBER.
E0529 13:43:40.369395592     212 ssl_transport_security.cc:1495] Handshake failed with fatal error SSL_ERROR_SSL: error:100000f7:SSL routines:OPENSSL_internal:WRONG_VERSION_NUMBER.
E0529 13:43:53.219741713     218 ssl_transport_security.cc:1495] Handshake failed with fatal error SSL_ERROR_SSL: error:100000f7:SSL routines:OPENSSL_internal:WRONG_VERSION_NUMBER.
E0529 13:44:07.136260220     212 ssl_transport_security.cc:1495] Handshake failed with fatal error SSL_ERROR_SSL: error:100000f7:SSL routines:OPENSSL_internal:WRONG_VERSION_NUMBER.
E0529 13:44:24.241514722     218 ssl_transport_security.cc:1495] Handshake failed with fatal error SSL_ERROR_SSL: error:100000f7:SSL routines:OPENSSL_internal:WRONG_VERSION_NUMBER.
E0529 13:44:43.304245898     218 ssl_transport_security.cc:1495] Handshake failed with fatal error SSL_ERROR_SSL: error:100000f7:SSL routines:OPENSSL_internal:WRONG_VERSION_NUMBER.
E0529 13:45:11.845964912     218 ssl_transport_security.cc:1495] Handshake failed with fatal error SSL_ERROR_SSL: error:100000f7:SSL routines:OPENSSL_internal:WRONG_VERSION_NUMBER.
E0529 13:45:54.718151462     212 ssl_transport_security.cc:1495] Handshake failed with fatal error SSL_ERROR_SSL: error:100000f7:SSL routines:OPENSSL_internal:WRONG_VERSION_NUMBER.
E0529 13:46:22.455457989     218 ssl_transport_security.cc:1495] Handshake failed with fatal error SSL_ERROR_SSL: error:100000f7:SSL routines:OPENSSL_internal:WRONG_VERSION_NUMBER.
```

**Comment by yanliang567** [comment]
/assign @XuanYang-cn 
I think it was caused by secure=True, could you please help to double check and fix it?
/unassign

**Comment by NicoYuan1986** [comment]
Update: https://jenkins.milvus.io:18080/blue/organizations/jenkins/Milvus%20Nightly%20CI/detail/2.2.0/207/pipeline/188

1. log: [artifacts-milvus-distributed-kafka-nightly-207-pymilvus-e2e-logs.tar.gz](https://jenkins.milvus.io:18080/job/Milvus%20Nightly%20CI/job/2.2.0/207/artifact/artifacts-milvus-distributed-kafka-nightly-207-pymilvus-e2e-logs.tar.gz)
2. failed time: [2023-05-31T20:07:21.667Z] [gw1] [ 11%] FAILED testcases/test_connection.py::TestConnect::test_parameters_with_uri_connection[default-https] 
```
[2023-05-31T21:20:22.195Z] ----------------------------- Captured stderr call -----------------------------
[2023-05-31T21:20:22.195Z] E0531 20:07:10.579582694   23040 ssl_transport_security.cc:1420]       Handshake failed with fatal error SSL_ERROR_SSL: error:100000f7:SSL routines:OPENSSL_internal:WRONG_VERSION_NUMBER.
[2023-05-31T21:20:22.195Z] E0531 20:07:11.574628056   23040 ssl_transport_security.cc:1420]       Handshake failed with fatal error SSL_ERROR_SSL: error:100000f7:SSL routines:OPENSSL_internal:WRONG_VERSION_NUMBER.
[2023-05-31T21:20:22.195Z] E0531 20:07:13.492389038   23040 ssl_transport_security.cc:1420]       Handshake failed with fatal error SSL_ERROR_SSL: error:100000f7:SSL routines:OPENSSL_internal:WRONG_VERSION_NUMBER.
[2023-05-31T21:20:22.195Z] E0531 20:07:15.577355195   23040 ssl_transport_security.cc:1420]       Handshake failed with fatal error SSL_ERROR_SSL: error:100000f7:SSL routines:OPENSSL_internal:WRONG_VERSION_NUMBER.
[2023-05-31T21:20:22.195Z] E0531 20:07:19.611383485   23040 ssl_transport_security.cc:1420]       Handshake failed with fatal error SSL_ERROR_SSL: error:100000f7:SSL routines:OPENSSL_internal:WRONG_VERSION_NUMBER.
[2023-05-31T21:20:22.195Z] ------------------------------ Captured log call -------------------------------
[2023-05-31T21:20:22.195Z] [2023-05-31 20:07:10 - DEBUG - ci_test]: (api_request)  : [Connections.connect] args: ['default', '', '', ''], kwargs: {'uri': 'https://mdk-207-n-milvus.milvus-ci:19530'} (api_request.py:56)
[2023-05-31T21:20:22.195Z] [2023-05-31 20:07:20 - ERROR - ci_test]: Traceback (most recent call last):
[2023-05-31T21:20:22.195Z]   File "/usr/local/lib/python3.8/site-packages/pymilvus/client/grpc_handler.py", line 119, in _wait_for_channel_ready
[2023-05-31T21:20:22.195Z]     grpc.channel_ready_future(self._channel).result(timeout=timeout)
[2023-05-31T21:20:22.195Z]   File "/usr/local/lib/python3.8/site-packages/grpc/_utilities.py", line 151, in result
[2023-05-31T21:20:22.195Z]     self._block(timeout)
[2023-05-31T21:20:22.195Z]   File "/usr/local/lib/python3.8/site-packages/grpc/_utilities.py", line 97, in _block
[2023-05-31T21:20:22.195Z]     raise grpc.FutureTimeoutError()
[2023-05-31T21:20:22.195Z] grpc.FutureTimeoutError
[2023-05-31T21:20:22.195Z] 
[2023-05-31T21:20:22.195Z] The above exception was the direct cause of the following exception:
[2023-05-31T21:20:22.195Z] 
[2023-05-31T21:20:22.195Z] Traceback (most recent call last):
[2023-05-31T21:20:22.195Z]   File "/home/jenkins/agent/workspace/tests/python_client/utils/api_request.py", line 26, in inner_wrapper
[2023-05-31T21:20:22.195Z]     res = func(*args, **_kwargs)
[2023-05-31T21:20:22.195Z]   File "/home/jenkins/agent/workspace/tests/python_client/utils/api_request.py", line 57, in api_request
[2023-05-31T21:20:22.195Z]     return func(*arg, **kwargs)
[2023-05-31T21:20:22.195Z]   File "/usr/local/lib/python3.8/site-packages/pymilvus/orm/connections.py", line 349, in connect
[2023-05-31T21:20:22.195Z]     connect_milvus(**kwargs, user=user, password=password, token=token, db_name=db_name)
[2023-05-31T21:20:22.195Z]   File "/usr/local/lib/python3.8/site-packages/pymilvus/orm/connections.py", line 282, in connect_milvus
[2023-05-31T21:20:22.195Z]     gh._wait_for_channel_ready(timeout=timeout)
[2023-05-31T21:20:22.195Z]   File "/usr/local/lib/python3.8/site-packages/pymilvus/client/grpc_handler.py", line 123, in _wait_for_channel_ready
[2023-05-31T21:20:22.195Z]     raise MilvusException(Status.CONNECT_FAILED,
[2023-05-31T21:20:22.195Z] pymilvus.exceptions.MilvusException: <MilvusException: (code=2, message=Fail connecting to server on mdk-207-n-milvus.milvus-ci:19530. Timeout)>
[2023-05-31T21:20:22.195Z]  (api_request.py:39)
[2023-05-31T21:20:22.195Z] [2023-05-31 20:07:20 - ERROR - ci_test]: (api_response) : <MilvusException: (code=2, message=Fail connecting to server on mdk-207-n-milvus.milvus-ci:19530. Timeout)> (api_request.py:40)
```

**Comment by XuanYang-cn** [comment]
@NicoYuan1986 Is this still a problem? I believe we've fixed some secured=True problem in pymilvus 2.2.9

**Comment by NicoYuan1986** [comment]
Not reproduced for weeks.
